### DISCUSSION TITLE
be 扩缩容失败，分片均衡REDUNDANT 类型，大量无法执行，ErrMsg:wait txn before 373885 to be finished

### DISCUSSION THREAD

**Discussion by zjh497463046** [apache/doris#15720] [A - General / Q&A]
原系统版本，1.0.0，当操作扩容be后执行缩容操作，执行SHOW PROC '/cluster_balance/pending_tablets';
可以看到大量，State:REDUNDANT,持续执行且无法成功，错误信息：wait txn before 373885 to be finished
因此升级了be和fe至1.1.4，因为问题出现在生产环境，uat我们验证了扩缩容成功后。
在1.1.4版本此问题依然出现，尝试过提高修复表优先级，停止均衡在恢复均衡，修改各类配置参数，强制下线be，都无法因此此任务
拿一分片举例
10002-10005都是希望下线的，其中10003和10004执行了强制下线，可以看到分片状态依旧不对
![image](https://user-images.githubusercontent.com/19945318/211236899-356f533e-c67c-4a88-8013-c520a8f9738e.png)

后台fe日志可以看到下线的信息，但无报错信息，此信息重复执行
![image](https://user-images.githubusercontent.com/19945318/211237000-48df77c9-9f5a-4467-9efa-1518e00e2c95.png)

**Comment by zjh497463046** [comment]
补充SHOW PROC '/cluster_balance/pending_tablets';截图
![image](https://user-images.githubusercontent.com/19945318/211237109-5bd838a1-c0d2-4ce5-b971-890a68ca820e.png)
![image](https://user-images.githubusercontent.com/19945318/211237122-8b4c4856-48e9-43c6-ae27-f455990a3b9b.png)
补充show proc "/cluster_health/tablet_health";截图
![image](https://user-images.githubusercontent.com/19945318/211237151-9d5d3d42-fe76-47be-b300-2703471708ec.png)
补充SHOW PROC '/backends';截图
![image](https://user-images.githubusercontent.com/19945318/211237264-94c4a32e-2b36-4abd-8c96-2fa0b7862974.png)

**Comment by Power098** [comment]
我们2.1.3版本也出现了相同的问题，请问这个问题是怎么解决的

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
